components/broadcast_talent_table/commons/broadcast_talent_table.lua

```diff
@@ -0,0 +1,390 @@
+end
+
+function BroadcastTalentTable:_header()
+	local header = mw.html.create('tr')
+		:tag('th'):wikitext('Date'):css('width', '120px'):done()
+		:tag('th'):wikitext('Tier'):css('width', '50px'):done()
+		:tag('th'):wikitext('Tournament')
+			:attr('colspan', self.args.displayGameIcon and 3 or 2)
+			:css('width', self.args.displayGameIcon and '350px' or '300px'):done()
+		:tag('th'):wikitext('Position'):css('width', '130px'):done()
+
+	if not self.args.displayPartnerListColumn then
+		return header
+	end
+
+	return header:tag('th'):wikitext('Partner List'):css('width', '160px'):done()
+end
+
+function BroadcastTalentTable._seperator(seperatorTitle)
+	return mw.html.create('tr'):addClass('sortbottom'):css('font-weight', 'bold')
+		:tag('td'):attr('colspan', 42):wikitext(seperatorTitle):done()
+end
+
+function BroadcastTalentTable:_row(tournament)
+	local row = mw.html.create('tr')
+
+	tournament = BroadcastTalentTable._fetchTournamentData(tournament)
+
+	if HighlightConditions.tournament(tournament, self.args) then
+		row:addClass('tournament-highlighted-bg')
+	end
+
+	row
+		:tag('td'):wikitext(tournament.date):done()
+		:tag('td'):wikitext(self:_tierDisplay(tournament)):done()
+
+	if self.args.displayGameIcon then
+		row:tag('td'):node(Game.icon{game = tournament.game})
+	end
+
+	row
+		:tag('td'):wikitext(LeagueIcon.display{
+			icon = tournament.icon,
+			iconDark = tournament.icondark,
+			series = tournament.series,
+			date = tournament.date,
+			link = tournament.pagename,
+			name = tournament.name,
+		}):done()
+		:tag('td'):wikitext(Page.makeInternalLink({},
+			BroadcastTalentTable._tournamentDisplayName(tournament),
+			tournament.pagename
+		)):done()
+		:tag('td'):wikitext(table.concat(tournament.positions, '<br>')):done()
+
+	if not self.args.displayPartnerListColumn then
+		return row
+	end
+
+	return row:tag('td'):node(self:_partnerList(tournament)):done()
+end
+
+function BroadcastTalentTable._fetchTournamentData(tournament)
+	local queryData = mw.ext.LiquipediaDB.lpdb('tournament', {
+		conditions = '[[pagename::' .. tournament.parent .. ']]',
+		query = 'name, tickername, icon, icondark, series, game, '
+			.. 'liquipediatier, liquipediatiertype, publishertier, extradata',
+	})
+
+	if type(queryData[1]) ~= 'table' then
+		return tournament
+	end
+
+	queryData[1].tournamentExtradata = queryData[1].extradata
+
+	return Table.merge(queryData[1], tournament)
+end
+
+function BroadcastTalentTable._tournamentDisplayName(tournament)
+	local extradata = tournament.extradata or {}
+	if Logic.readBool(extradata.showmatch) and String.isNotEmpty(extradata.showmatchname) then
+		return extradata.showmatchname
+	end
+
+	local displayName = String.isNotEmpty(tournament.tickername) and tournament.tickername
+		or String.isNotEmpty(tournament.name) and tournament.name
+		or tournament.parent:gsub('_', ' ')
+
+	if not Logic.readBool(extradata.showmatch) then
+		return displayName
+	end
+
+	return displayName .. ' - Showmatch'
```

Comment:
Why would a tournament store `showmatch(name)` in extradata instead of just updating the (ticker)name?
Would make more sense to me if it was on a placement extradata, but tournament?

Reply:
tbh i do not understand why anyone would do it either
i just ported it from the currently used version to not break shit

will do some (insource) searches if it is actually used somewhere

Reply:
nothing stores it at all ...
will kick accordingly

Reply:
reverted the kick and added some comments

Reply:
but the module that stores it still needs to set it in extradata hence it should find that module

Reply:
> why the hell did https://tools.liquipedia.space/?view=search not find that ...

Doubled checked now, worked for me

Reply:
#badLuck for me^^